device/RPI.conf

```
@@ -1,34 +1,34 @@
 # https://www.raspberrypi.org/products/raspberry-pi-3-model-b/
 # https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/
+# https://www.raspberrypi.com/products/raspberry-pi-4-model-b/
 
 export MAKE_ARGS_DEV="
```

```suggestion
export MAKE_ARGS_DEV="
CROSS_BINUTILS_PREFIX=/usr/local/aarch64-unknown-freebsd${SRCREVISION}/bin
```

put this back for now to defer to other PR?

---

Done